src/connectors/kokkos/kokkos-connector.cpp

```diff
@@ -491,7 +492,7 @@ extern "C" void kokkosp_request_values(
   auto iter =
       tuning_regions.emplace(tuningProblem, std::make_pair("", nullptr));
   if (iter.second) {
-    std::string prefix = "dtree-step-0-rank-0-";
+    std::string prefix = "RandomForest-step-0-rank-0-";
```

Comment:
Lemme test these locally, I'll let you know if it works out

Reply:
Thanks! I was trying to add you as a reviewer but your name doesn't appear in the list. Any idea why?